apps/src/templates/manageStudents/MoveStudents.jsx

```diff
@@ -249,13 +244,20 @@ class MoveStudents extends Component {
     });
   };
 
+  isMovableSection = (section) => {
+    if (section.id === this.props.currentSectionId) { return false; }
+    if (section.loginType === SectionLoginType.google_classroom || section.loginType === SectionLoginType.clever) { return false; }
+    return true;
+  };
+
   renderOptions = () => {
-    const {sections, currentSectionId} = this.props;
-    let options = sections.map(section => {
-      if (section.id === currentSectionId) {
-        return null;
-      } else {
+    const {sections} = this.props;
+    let options = Object.keys(sections).map(sectionId => {
+      const section = sections[sectionId];
+      if (this.isMovableSection(section)) {
         return <option key={section.id} value={section.id}>{section.name}</option>;
+      } else {
+        return null;
       }
```

Comment:
:dizzy_face: I understand that React renders this just fine, but it still bugs me to return `null` values in this array.  Any reason not to `.filter(x => x)`?

Reply:
+1 for `.filter(...)`